aspnetcore/signalr/background-service/samples/3.0/Clients.ConsoleTwo/ClockHubClient.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using HubServiceInterfaces;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Clients.ConsoleTwo
+{
+    public class ClockHubClient : IClock, IHostedService
+    {
+#region ClockHubClientCtor
```

Comment:
No regions

Reply:
Oh wait is this for the docs?

Reply:
The regions are no different to the original sample, do you want them removed? They're linked to in the related docs page.

Reply:
> Oh wait is this for the docs?

Yes :D

Reply:
Regions are needed to import snippets. This is a vast improvement to the previous version that pasted snippets into the MD file.

The preferred approach when more than 3 or 4 pair of monikers are used is  use monikers on continuous areas only:

```
::: moniker range=">= aspnetcore-3.0"

// Entire 3.1 version here

::: moniker-end
::: moniker range="< aspnetcore-3.0"

// Entire 2.1 version here

::: moniker-end
```

That makes it much easier to add the 5.0 and 5.1 versions. It's much easier to maintain the doc.